.github/ISSUE_TEMPLATE.md

```diff
@@ -4,3 +4,11 @@ If you're just making a suggestion, be descriptive, and link to any issues that
 
 You can delete from this line up.
 ---------------------
+
+* Panel or Daemon: 
+* Version of Panel/Daemon: 
+* Your Computer's or Server's OS/Browser: 
+* Issue Type (Bug/Question/Feature): 
```

Comment:
I think this (issue type) can generally be inferred by what the user is asking, seems redundant to add it.

I also think we could split the Computer/Server OS stuff into different lines.

```
* Server OS (include output of `uname -a`):
* Computer OS and Browser (if applicable):
```